source/reference/minio-mc/mc-stat.rst

```diff
@@ -91,17 +92,18 @@ Parameters
 
       mc stat ~/data/myobject.txt
 
-.. mc-cmd:: --encrypt-key
+.. mc-cmd:: --enc-c
```

Comment:
Was `encrypt-key` ever deprecated, or immediately replaced with `enc-c`? Guess we need `versionchanged` or `versionadded`

Reply:
Just immediately replaced everywhere. This isn't the only command that had the swap. It's just left over and I happened to catch it while reviewing the other new flags.

Reply:
I'm not even sure what the version was that changed it; this was a drive by fix.